setjmp/README.md

```diff
@@ -0,0 +1,79 @@
+# Long Jumps Falling Short - Control-Flow Tracking & Misuse Detection for Non-Local Jumps in C
+
+This page gives additional information on the experiments & implementation
+that accompanies our manuscript.
+
+# Basic Setup
+
+On top of this repository, the Goblint Static Analyzer is needed.
+The relevant tag is this one: https://github.com/goblint/analyzer/tree/v2.1.0-longjmp
+
+Please follow the installation guide given in https://github.com/goblint/analyzer/tree/v2.1.0-longjmp#readme
+
+# Small Litmus Tests
+## General
+
+The set of 40-ish small examples is integrated into the set of regression tests for the Goblint.
+To run all regression tests, one may simply (in the analyzer directory) call
+
+> make test
+
+In order to run one specific example, one may run
+
+> ./regtest.sh 66 NN
+
+where `NN` is the number at the beginning of the testcase in folder `tests/regression/66-longjmp/NN-one.c`.
```

Comment:
The group number will be changed due to renumbering. This won't affect the tagged thing, but will only keep the description up to date.